gsoc/2021.md

```diff
@@ -0,0 +1,505 @@
+﻿---
+layout: inner-page-no-masthead
+title: "Google Summer of Code 2021 Scala Projects"
+permalink: /gsoc/2021/
+includeTOC: true
+---
+
+## Summary & goals
+
+[The Scala Center](https://scala.epfl.ch/) organisation and its team are dedicated to moving the needle towards a more diverse community, (amongst other) by providing easy-to-access first experience and introduction to the Scala ecosystem. GSoC is a fantastic opportunity to engage with students all around the world and onboard new generations, as well as underrepresented groups in tech!
+
+
+Scala is a part of Google Summer of Code since 2010. With 65 projects & students, ~70 mentors over the years, many of whom continued contributing to Scala, we are happy to announce that in 2021 the Scala Center will reignite the collaboration and commit to providing an awesome experience going forward.
+
+These are especially exciting times: Scala 3-RC1 has been released mid-February 2021, after 8 years of development. We invite you to chose your project and enjoy the fresh new approach!
+
+## More about the history
+In the past [LAMP](https://www.epfl.ch/labs/lamp/), the lab behind the
+development of Scala at EPFL, invested a lot of time and effort into GSOC,
+mentoring an average of 8 students every year and having participated in 9
+editions of the program starting as early as 2010. In the last 2 editions a
+lack of human resources and the development of Scala 3 prevented Scala from
+participating in GSOC.
+
+The Scala Center, a not-for-profit organization with the mission of open
+source and education, decided to take over and restart the long tradition of
+students entering our community thanks to GSOC. We collected as much
+experience and advice as possible from LAMP and from both students and
+mentors from previous editions and we are organizing the 2021 edition.
+
+We hope that our project list will inspire a motivated, diverse and
+open-minded team of students to become part of our community by spending a
+few months working on major open source projects focusing on topics such as
+developer tooling, data science, concurrent computing, metaprogramming, user
+interfaces, education and functional programming.
+
+### Resources
+
+- Find out who we are on the [Scala Center website](https://scala.epfl.ch/)
+- To know more about Google Summer of Code, please visit [the website of the event](https://summerofcode.withgoogle.com/).
+
+## Table of Contents
+
+ - [scala-native-bindgen](#scala-native-bindgen)
+ - [Make scalajs-jfe production-ready](#Make-scalajs-jfe-production-ready)

```

Comment:
all the `#` links should be lowercased, otherwise they will not work I think.

Reply:
It works [here](https://github.com/scalacenter/GoogleSummerOfCode2021/blob/main/README.md)

Reply:
I don't thinks so. It didn't work locally, and lowercase did.

Reply:
Even moving to lowercase (see my last commit) does not fix my local build.

Reply:
That is a separate issue.